nsv13/code/game/objects/items/bloody_eye.dm

```diff
@@ -0,0 +1,196 @@
+	reagent_state = LIQUID
+	color = "#D92323"
+	overdose_threshold = 5
+	addiction_threshold = 5
+	metabolization_rate = 0.75 * REAGENTS_METABOLISM
+
+/datum/reagent/drug/bloody_eye/on_mob_metabolize(mob/living/L)
+	ADD_TRAIT(L,TRAIT_MONKEYLIKE,type)
+	ADD_TRAIT(L,TRAIT_NOCRITDAMAGE,type)
+	ADD_TRAIT(L,TRAIT_NOSOFTCRIT,type)
+	ADD_TRAIT(L,TRAIT_NOHARDCRIT,type)
+	ADD_TRAIT(L,TRAIT_NOBLOCK,type)
+	ADD_TRAIT(L,TRAIT_STUNIMMUNE,type)
+	ADD_TRAIT(L,TRAIT_SLEEPIMMUNE,type)
+	ADD_TRAIT(L,TRAIT_IGNOREDAMAGESLOWDOWN,type)
+	ADD_TRAIT(L,TRAIT_NOSTAMCRIT,type)
+	ADD_TRAIT(L,TRAIT_NOLIMBDISABLE,type)
+	L.apply_status_effect(/datum/status_effect/bloody_eye)
+	..()
+	if (L.client)
+		SSmedals.UnlockMedal(MEDAL_APPLY_REAGENT_METH,L.client)
+
+	L.add_movespeed_modifier(type, update=TRUE, priority=100, multiplicative_slowdown=-2, blacklisted_movetypes=(FLYING|FLOATING))
+
+//You won't go down unless you get killed with excess damage.
+
+/datum/reagent/drug/bloody_eye/on_mob_end_metabolize(mob/living/L)
+	REMOVE_TRAIT(L,TRAIT_MONKEYLIKE,type)
+	REMOVE_TRAIT(L,TRAIT_NOCRITDAMAGE,type)
+	REMOVE_TRAIT(L,TRAIT_NOSOFTCRIT,type)
+	REMOVE_TRAIT(L,TRAIT_NOHARDCRIT,type)
+	REMOVE_TRAIT(L,TRAIT_NOBLOCK,type)
+	REMOVE_TRAIT(L,TRAIT_STUNIMMUNE,type)
+	REMOVE_TRAIT(L,TRAIT_SLEEPIMMUNE,type)
+	REMOVE_TRAIT(L,TRAIT_IGNOREDAMAGESLOWDOWN,type)
+	REMOVE_TRAIT(L,TRAIT_NOSTAMCRIT,type)
+	REMOVE_TRAIT(L,TRAIT_NOLIMBDISABLE,type)
+	L.remove_status_effect(/datum/status_effect/bloody_eye)
+	L.remove_movespeed_modifier(type)
+	..()
+
+/datum/reagent/drug/bloody_eye/on_mob_life(mob/living/carbon/M)
+	var/high_message = pick("You see red.", "You feel like ripping out someone's throat.", "You feel like nothing could ever kill you.")
+	if(prob(5))
+		to_chat(M, "<span class='notice'>[high_message]</span>")
+	M.AdjustStun(-40, FALSE)
+	M.AdjustKnockdown(-40, FALSE)
+	M.AdjustUnconscious(-40, FALSE)
+	M.AdjustParalyzed(-40, FALSE)
+	M.AdjustImmobilized(-40, FALSE)
+	M.adjustStaminaLoss(-30, 0)
+	M.Jitter(2)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 0.25)
+	if(prob(5))
+		M.emote(pick("scream", "laugh"))
+	..()
+	. = 1
+
+/datum/reagent/drug/bloody_eye/overdose_start(mob/living/M)
+	to_chat(M, "<span class='userdanger'>KILLKILLKILLKILLKILLKILLKILLKILL!</span>")
+	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "[type]_overdose", /datum/mood_event/overdose, name)
+
+/datum/reagent/drug/bloody_eye/overdose_process(mob/living/M)
+	if((M.mobility_flags & MOBILITY_MOVE) && !ismovableatom(M.loc))
+		for(var/i in 1 to 4)
+			step(M, pick(GLOB.cardinals))
+	if(prob(20))
+		M.emote("scream")
+	if(prob(33))
+		M.visible_message("<span class='danger'>[M]'s eyes are blood red!</span>")
+		M.drop_all_held_items()
+	..()
+	M.adjustToxLoss(0.25, 0)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, pick(0.1, 0.15, 0.2, 0.25))
+	. = 1
+
+/datum/reagent/drug/bloody_eye/addiction_act_stage1(mob/living/M)
+	M.Jitter(20)
+	if(prob(20))
+		M.emote(pick("scream","laugh"))
```

Comment:
ditto

Reply:
The effects are supposed to get worse over time and the way drugs do ticks versus the way status effects work, it works better as is for what I want it do.

Reply:
ditto meaning "you should also make this emote an async call"

Reply:
Oh that's what you mean, got it. Easy easy.